### PR TITLE
Allow enter on form submit

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@secretrobotron R?

**Comment by alicoding** [comment]
Sorry for using this patch based on another commit that's not landed @secretrobotron but it's a bit dependent here...

File: templates/components/form/form.jsx [hunk 1]
```diff
@@ -160,7 +160,10 @@ var Form = React.createClass({
   },
   render: function() {
      var fields = Object.keys(this.props.fields).map(this.buildFormElement);
-     return <div role="form">{fields}</div>;
+     return <div role="form"><form action="#" onSubmit={this.processFormData} id="form">{fields}<input className="hidden" type="submit"/></form></div>;
+  },
+  onSubmit: function(e) {
+    console.log('here')
```
Comment (secretrobotron) [inline templates/components/form/form.jsx]: Remove this¡

**Comment by alicoding** [comment]
@cadecairos R? too if you can

**Comment by cadecairos** [comment]
R+

**Comment by secretrobotron** [comment]
@alicoding what other patch is this based on? Need to rebase?

**Pull request merged**